fixture/db/client.json

```diff
@@ -19,7 +19,7 @@
             "client_secret_expires_at": 0,
             "client_secret": "VVrUT7n82KNcuh947RAI-dqY79K8r47k_Xk9WvdKXl_viltV5kzX5oNDjen97vr0pBRSlv3oGZvz3_ka1136LQ",
             "redirect_uris": ["https://localhost:8082/signin-oidc"],
-            "scope": "openid pools:read pools:write withdrawals:read rewards:write promotions:read promotions:write deposits:write deposits:read"
+            "scope": "openid rewards:read erc20:read erc721:read withdrawals:read withdrawals:write deposits:read deposits:write account:read account:write memberships:read memberships:write promotions:read relay:write"
```

Comment:
@GarfDev Could you explain the changes you made here? It seems not ok to me that pool permissions are removed and a relay permission is added.

Reply:
I have copied changes from oicd.ts of dashboard into here

Reply:
Looking at the specific scopes I think you have mistakenly copied the scopes for the wallet in here! Please verify that:)